test/algorithms/ground_state_solvers/test_adapt_vqe.py

```diff
@@ -69,6 +74,72 @@ def test_default(self):
         res = calc.solve(self.problem)
         self.assertAlmostEqual(res.electronic_energies[0], self.expected, places=6)
 
+    def test_param_shift(self):
```

Comment:
We could group most of these tests into one using `data` and `ddt` and avoid boilerplate code 😄 
```python
from ddt import ddt, data

@ddt
class TestAdaptVQE(...):

    @data("param_shift", "fin_diff", "lin_comb")
    def test_gradient(self, grad_method):
        grad = Gradient(grad_method)
        ...
```
(we'd have to add a bit more to also pass `epsilon=1` for finite differences)